core/src/test/scala/org/genivi/sota/core/CampaignResourceSpec.scala

```diff
@@ -323,4 +355,74 @@ class CampaignResourceSpec extends FunSuite
 
     getCampaignsOk().filter(_.id == id).map(_.createdAt) shouldEqual createdAt
   }
+
+  test("campaign with static delta changes to 'in preparation' after launch") {
```

Comment:
Made the function to cut down on duplicate code, then forget to actually use it :/